srcpkgs/itch/template

```diff
@@ -0,0 +1,44 @@
+# Template file for 'itch'
+pkgname=itch
+version=1.24.0
+revision=1
+archs="x86_64"
+wrksrc="${pkgname}-setup-${version}"
+build_style=go
+go_import_path="github.com/itchio/itch-setup"
+hostmakedepends="git pkg-config nodejs-lts"
+makedepends="gtk+3-devel"
+short_desc="Itch.io indie games client for Linux"
+maintainer="MarcoAPC <marcoaureliopc@gmail.com>"
+license="MIT"
+homepage="https://itch.io/"
+distfiles="
+ https://github.com/itchio/itch-setup/archive/v${version}.tar.gz
+ https://raw.githubusercontent.com/itchio/itch/master/release/images/itch-icons/icon16.png
+ https://raw.githubusercontent.com/itchio/itch/master/release/images/itch-icons/icon32.png
+ https://raw.githubusercontent.com/itchio/itch/master/release/images/itch-icons/icon48.png"
```

Comment:
Please use the icons from version instead of master

```suggestion
 https://raw.githubusercontent.com/itchio/itch/v${version}/release/images/itch-icons/icon16.png
 https://raw.githubusercontent.com/itchio/itch/master/release/images/itch-icons/icon32.png
 https://raw.githubusercontent.com/itchio/itch/master/release/images/itch-icons/icon48.png"
```

Reply:
It will stay like this! 

 https://raw.githubusercontent.com/itchio/itch/v25.4.1/release/images/itch-icons/icon16.png
 https://raw.githubusercontent.com/itchio/itch/v25.4.1/release/images/itch-icons/icon32.png
 https://raw.githubusercontent.com/itchio/itch/v25.4.1/release/images/itch-icons/icon48.png

I'm using the framework icons! 

https://github.com/itchio/itch

The compiled app does not have these icons! 

https://github.com/itchio/itch-setup

Reply:
Ok, you can use a specific tag from that repo. I just want to avoid pulling from the development branch, since any change they make will break our build.

Reply:
Done!